crates/symbolicator/Cargo.toml

```diff
@@ -43,7 +43,7 @@ serde = { version = "1.0.119", features = ["derive", "rc"] }
 serde_json = "1.0.61"
 serde_yaml = "0.8.15"
 structopt = "0.3.21"
-symbolic = { git = "https://github.com/getsentry/symbolic", branch = "fix/demangle-fixes", version = "8.2.0", features = ["common-serde", "debuginfo", "demangle", "minidump-serde", "symcache"] }
+symbolic = { git = "https://github.com/getsentry/symbolic", branch = "fix/demangle-fixes", version = "8.2.1", features = ["common-serde", "debuginfo", "demangle", "minidump-serde", "symcache"] }
```

Comment:
While we're on the branch, you can remove the version here.

Reply:
I know, but think its a good idea to keep it actually. Specifically, I bumped the version here, and `cargo update` reminded me that I forgot to forward that branch. That also reminds me that I want to get rid of that at some point 🤦🏻‍♂️